src/foundry/client/apps/forms/journal-page-sheet.d.ts

```diff
@@ -0,0 +1,355 @@
+import type { EditorView } from "prosemirror-view";
+import type Showdown from "showdown";
+import type { Editor } from "tinymce";
+import type { CONST } from "../../../common/module.mjs.js";
+
+declare global {
+  /**
+   * The Application responsible for displaying and editing a single {@link JournalEntryPage} document.
+   *
+   * @typeParam ConcreteOptions - The type of the options object
+   */
+  class JournalPageSheet<
+    ConcreteOptions extends JournalPageSheet.Options = JournalPageSheet.Options
+  > extends DocumentSheet<ConcreteOptions, JournalEntryPage> {
```

Comment:
we discussed this on discord, but in the interest of preserving it for posterity:

these were intentionally omitted. in FVTT's v10.288 source code, the **only** classes in this PR that have explicit `@param` annotations on their class to describe their constructor are `BaseJournalEntry` and `BaseJournalEntryPage`. 

the annotations that *do* exist don't provide meaningful new information. here's an example of what i mean -- compare `BaseJournalEntry` `@param`s with the corresponding annotation it would otherwise inherit from `Document`:

<img width="862" alt="Screen Shot 2022-10-15 at 11 23 19 PM" src="https://user-images.githubusercontent.com/5354757/196021389-5bb5c486-c9e1-4fee-8804-e0addf1c49ac.png">
<img width="809" alt="Screen Shot 2022-10-15 at 11 28 34 PM" src="https://user-images.githubusercontent.com/5354757/196021570-0487c031-e1ea-427f-b374-7ebe2d785ddd.png">

fair enough if it's easier in the long run to just keep it standardized with the original source code where possible. i've added constructor annotations where explicit class `@param` annotation is present.